radsat-sk/operation/subsystems/RAntenna.c

```diff
@@ -282,3 +284,27 @@ int antennaTelemetry(antenna_telemetry_t* telemetry) {
 
 	return 0;
 }
+
+/**
+ * Resets microcontroller of the antenna
+ *
+ *  @return 0 for success, non-zero for failure. See hal/errors.h for details.
+ */
+int antennaReset(void){
+
+	// Reset both side A/B antennas. See section 6.2 of Antenna System User Manual
+
+	int errorA = IsisAntS_reset(ANTENNA_I2C_SLAVE_ADDR_PRIMARY, isisants_sideA);  //reset on side A
+	int errorB = IsisAntS_reset(ANTENNA_I2C_SLAVE_ADDR_PRIMARY, isisants_sideB);  //reset on side B
```

Comment:
I'm very curious about the first argument here, not only in your code but in the rest of the document. You're providing the antenna slave address, but what it's actually asking for is the index. During initialization (see line 48 of `RAntenna.c`) we provide a pair of slave addresses, and the number "4" for "number of antennas" but I believe this needs to be reconsidered.

It seems to me like they're actually asking for the index in both cases, in the sense that the module is asking for "out of all the antennas on your system, which one do you want to talk to". I believe that since we only have the 1 antenna, the `index` or `number` argument should always be `0` or the enum `isisants_antenna1`. We can talk more about this in person, especially with Addi.